docs/faq-troubleshooting.md

```diff
@@ -0,0 +1,153 @@
+
+If this documentation was not enough, you could also try:
+
+* [open an issue](https://github.com/src-d/sourced-ce/issues), if you want to
+suggest a new feature, if you need assistance with a contribution, or if you
+found any bug.
+* [Visit the source{d} Forum](https://forum.sourced.tech) where users and community
+members discuss anything source{d} related. You will find there some common questions
+from other source{d} users, or ask yours.
+* [join our community on Slack](https://sourced-community.slack.com/join/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM),
+and talk with some of our engineers.
+
+
+## How Can I Update My Version Of source{d} CE?
+
+When there is a new release of **source{d} CE**, it is noticed every time a `sourced`
+command is called. When it happens you can download the new version from
+[src-d/sourced-ce/releases/latest](https://github.com/src-d/sourced-ce/releases/latest),
+and proceed as it follows:
+
+(You can also follow these steps if you want to update to any beta version, to
+downgrade, or to use your own built version of **source{d} CE**)
+
+1. download the new binary for your OS
+1. move the binary to the right place ([see Quickstart. Install](quickstart/2-install-sourced.md))
+1. run `sourced compose download`
+1. run `sourced restart` to apply the new configuration.
+
+This process will reinstall **source{d} CE** with the new components, but it will
+keep your data (repositories, charts, dashboards, etc). If you want to replace
+all the current dashboards with the ones from the release that you just installed,
+you have two alternatives:
+
+- run `sourced prune --all` before running `sourced init`; but if you were using
+repositories downloaded from a GitHub organization, they will be deleted, and
+downloaded again.
+- drop only the dashboards, and load the new ones, following also this other
+instructions: [How To Restore Dashboards and Charts to Defaults](#how-to-restore-dashboards-and-charts-to-defaults)
+
+
+## How To Restore Dashboards and Charts to Defaults
+
+In some circumstances, you may want to restore the UI dashboard and charts to its
+defaults. Following these steps, all the changes that you did in charts and
+dashboards will be replaced by the default ones for the version of **source{d} CE**
+that you're currently using. If you're using repositories from a GitHub organization,
+all its data will be preserved, and only charts and dashboards will be restarted.
+
+To do so, you only need to delete the docker volume containing the PostgreSQL
+database, and restart **source{d} CE**. It can be doing following these steps if
+you already have [Docker Compose](https://docs.docker.com/compose/) installed:
+
+```shell
+$ cd ~/.sourced/workdirs/__active__
+$ source .env
+$ docker-compose stop postgres
+$ docker-compose rm -f postgres
```

Comment:
it would destroy not only changes in dashboards but any other changes that were made by a user (new charts that don't belong to default dashboards, saved sql queries, ...). Maybe the better approach here would be to re-run import. It should in theory replace default dashboards but nothing else would be affected.

Reply:
@smacker good suggestions.
The import of the default dashboards (and default charts) is something that can be done running a command?  Could you help me to find where is it explained or documented?
If it's not a simple thing, I'd merge as it is now, adding your explanation:
```md
In some circumstances, you may want to restore the UI dashboard and charts to its
defaults. Following these steps, all the changes that you did in charts and
dashboards, plus your new charts, dashboards or saved queries that you created,
will be replaced by the default ones for the version of **source{d} CE** that
you're currently using.
```
then create an issue, and improve this later when we have this "restore" feature.

Reply:
clarified by d06a6df
Could we merge with this, and iterate later if there's no an easy way to reimport?

Reply:
```
docker-compose exec superset superset import_dashboards /correct/path
```

About the documentation to how to delete superset db completely. How is it better than just `prune` and `init` again?

Reply:
The clarification should be addressed by d06a6df

Reply:
```shell
docker-compose exec superset superset import_dashboard /home/superset/dashboards/metadata/overview.json
docker-compose exec superset superset import_dashboard /home/superset/dashboards/metadata/collaboration.json
```
The second one only if source{d} was initialized with orgs.

Also, if you `prune`, then you'll lost all the repos and metadata if initialized with orgs.

An alternative could be:
- `prune` and `init` if initialized with local,
- current docs if initialized with orgs.

I don't like offering two different ways to do it, but would you prefer this another way to explain it?

Reply:
My main concern that the question is how to re-create dashboards. But the answer is how to delete all the state. In the doc you mentioned only things I pointed to that would be deleted. But there are more:
- users
- roles
- logs
- favorites
- access levels
- ...most probably some more...

Reply:
Ok... next.
What about this other way?
>_In some circumstances, you may want to restore the UI dashboard and charts to its defaults. Following these steps, all the changes that you did in the UI, like in charts, dashboards,  saved queries, users, roles, etcetera, will be replaced by the default ones for the version of **source{d} CE** that you're currently using._

Or any other way to confirm that "_everything but repos and metadata will be deleted_" (we could also say it that way if you prefer)

Or also:
>_In some circumstances, you may want to restore the UI dashboard and charts to its
defaults. Currently, there is no clean way of doing it without deleting all the state of the UI._
>
>_Following these steps, all the state modified from the UI (charts, dashboards, saved queries, users, roles, etcetera) will be replaced by the default ones for the version of **source{d} CE**
that you're currently using. If you're using repositories from a GitHub organization,
all its data will be preserved, and only charts and dashboards will be restarted._
>
>_To do so, you only need to delete the docker volume containing the PostgreSQL
database, and restart **source{d} CE**. It can be done following these steps if
you already have [Docker Compose](https://docs.docker.com/compose/) installed:_

I think it's clear that **everything is going to be deleted**